### ISSUE TITLE
OWL Entity Description Editor view: On-the-fly entity creation is not working

### ISSUE THREAD

**Issue by csnyulas** [protegeproject/webprotege#595]
In earlier versions it was possible to create entities on-the fly in the OWL Entity Description Editor, but this is not possible any more. Maybe this is related to the fact that the whole context sensitive editing seems to be disabled.

**Comment by matthewhorridge** [comment]
Works for me...

<img width="601" alt="image" src="https://user-images.githubusercontent.com/488607/51290932-2eb90680-19ba-11e9-8bd5-79ddad296cf8.png">

**Comment by csnyulas** [comment]
@matthewhorridge you are right that it works in the "Entity Editor" view. The problem is in the "OWL Entity Description Editor" view, where it still does not work:

![image](https://user-images.githubusercontent.com/2441046/51307816-76a94f00-19f5-11e9-9d1c-c39f7bbbb940.png)

Note: "sour" and "sweet" was created on-the-fly in the "Entity Editor" view
![image](https://user-images.githubusercontent.com/2441046/51307975-e7506b80-19f5-11e9-9dc2-a4fdcb9aae84.png)
, and it was recognized in the "OWL Entity Description Editor" view and I could do auto-complete for them. However when I entered something like "bitter", which did not exist yet in the ontology, I did not get an option to create it on the fly.

**Comment by matthewhorridge** [comment]
Ah, right, sorry I was confused about this.  Thought it was the basic editor.
